tools/astarte_import/lib/mix/tasks/astarte_import.ex

```diff
@@ -5,6 +5,15 @@ defmodule Mix.Tasks.Astarte.Import do
   @impl Mix.Task
   @shortdoc "import data into an existing Astarte realm"
   def run(args) do
-    CLI.main(args)
+    IO.puts("Running astarte import task with arguments: #{inspect(args)}")
+
+    # Process the arguments
+    case args do
+      [realm, file_name] ->
+        CLI.main([realm, file_name])
```

Comment:
It would be enough to move the Supervision tree here, without using the `Application` module. This way, we don't mix in logic related to `Application`s, which are a bit different from CLI tools. My suggestion is then:
 ```suggestion
        Config.validate!()

        xandra_options = Config.xandra_options!()
        data_access_opts = [xandra_options: xandra_options]

        ae_xandra_opts = Keyword.put(xandra_options, :name, :xandra)

        children = [
          # Ensure the :astarte_data_access process is started
          {Xandra.Cluster, ae_xandra_opts},
          {Astarte.DataAccess, data_access_opts}
        ]

        opts = [strategy: :one_for_one, name: AstarteImport.Supervisor]
        Supervisor.start_link(children, opts)

        CLI.main([realm, file_name])
```

This would also remove the need for a `TaskRunner`, as the import would be executed directly in this file.